src/main/resources/static/css/styles.css

```diff
@@ -1,90 +1,77 @@
-@import url('https://fonts.googleapis.com/css?family=Montserrat:200,300,400,500,600,700');
-body, div, section, article, h2, header, footer{
-  box-sizing: border-box;
-  margin: 0px;
+* {
```

Comment:
Let's do away with the nested box look, it's off-putting and should be full-screen for a polished web app. 

I'd also say to pull the title of the app over onto the left side of the nav bar (hopefully in the next week we can made that the actual logo... I'm messing around with options for that). That'll give us a fairly simple layout that we can play with as we start adding things like the med log into the page.